lint.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 # If you pass arguments, this command will try to fix the lint
+files="epc-deck.js epc-graph.js epc-table.js epc-test.js epc-ui.js epc-code.js epc-polyfill.js"
 if [ $# -eq 0 ]
```

Comment:
I'm not super happy about checking for the presence of arguments, but then not looking at what those arguments are, for changing behavior.

How about this:

```
if [ "$1" = "--fix" ]
then
    eslint --fix $files
else
    eslint $files
fi
```

Reply:
Sure, I can change it. At some point though we should just move all of the files into a subdirectory so we don't need this script, and can just run `eslint src/` and have access to all of the other niceties their cli offers.